src/types.ts

```diff
@@ -567,6 +567,7 @@ export interface UnhashedOrder extends WyvernOrder {
   // OpenSea-specific
   makerReferrerFee: BigNumber
   waitingForBestCounterOrder: boolean
+  englishAuctionReservePrice?: BigNumber;
```

Comment:
in `test/api/api.ts`, let's start a unit test to test that orderToJSON is doing the right thing:
```

test('orderToJSON is correct', async () => {
    const accountAddress = ALEX_ADDRESS
    const takerAddress = ALEX_ADDRESS_2
    const amountInToken = 1.2
    const paymentTokenAddress = WETH_ADDRESS
    const expirationTime = Math.round(Date.now() / 1000 + 60) // one minute from now

    const tokenId = MYTHEREUM_TOKEN_ID.toString()
    const tokenAddress = MYTHEREUM_ADDRESS
    const order = await client._makeSellOrder({
      asset: { tokenAddress, tokenId },
      quantity: 1,
      accountAddress,
      startAmount: amountInToken,
      paymentTokenAddress,
      extraBountyBasisPoints: 0,
      buyerAddress: NULL_ADDRESS,
      expirationTime,
      waitForHighestBid: true,
      englishAuctionReservePrice: 2
    })

    const orderData = orderToJSON(order)
    assert.equal(...)
  })
```